### ISSUE TITLE
Stutter Occuring Every 3 Min 30 Sec of Play Time.

### ISSUE THREAD

**Issue by Pingore** [alvr-org/ALVR#1216]
## Description
A stutter is occurring consistently Every 3 Min 30 Sec. It goes in a wave like fashion where 1 or 2 frames get delayed, snowballs into full on stutter, then returns to normal. I have only been able to test it over USB as my router is far from my room. There is no stutter showing up on the Steam VR performance graph, and when I profiled the ALVR client nothing look out of the ordinary. I have tried many different configurations and none seem to improve this issue. It visually looks like frames are being played out of order, but it could also be holding frames (its hard to tell). This does not occur when I use link cable normally.

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)
- [x] I also have tried many different versions of ALVR (18.0.0, 18.2.3, 19.0.0 dev00, 19.0.0 dev04)

### Hardware

**CPU**: AMD Ryzen 5 3600

**GPU**: Nvidia GTX 1660 Super 6GB

**Audio**: Motherboard Audio (B450 Tomahawk) 

**USB Cable**: Oculus Link Cable

### Installation
**ALVR Version**: 18, 18.2.3, 19.0.0 dev00 (Nightly), 19.0.0 dev04 (Nightly),

**SteamVR Version**: 1.24

**Install Type**:
- [x] Packaged (`exe`, `deb`, `rpm`, etc)
- [x] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Version 21H2 (OS Build 19044.2005)

**Comment by Pingore** [comment]
I have updated my bug report after I did further testing today and found out that the rate in which this occurs is consistent (Every 3 Min 30 Sec). The start of it does not correlate to the start of a game.

**Comment by dogzz9445** [comment]
I had stutter a few days ago on ALVR v18.2.3. It was caused by network delay while using Wi-Fi. 
Network interference was caused by many Wi-Fi APs (more than 6 APs per channel). The first few minutes were fine, but some delays began to occur, and then there was a delay of more than a few seconds. Finally, black spots appeared on the Oculus screen and the screen paused. The PC display worked properly tracking the headset. I had to restart ALVR.
Although the above situation is not in general, I leave this comment one of the stuttering issues and that ALVR cannot recover alone.

**Comment by Pingore** [comment]
I should also mention that this issue tends to present itself mainly on 72hz. I went up to 80hz and it was still there but the duration was 1/5th of what it was before. Definitely isn't my router because I'm using a link cable.

**Comment by Pingore** [comment]
I investigated further and realized there is actually a correlation in the graph to the issue occurring. You can actually see the wave like lag occurring in the graph:
https://user-images.githubusercontent.com/94215310/198703478-842125c3-b6c1-4809-9622-8ab4536c8dee.mp4

**Comment by Pingore** [comment]
OK so i checked more in the headset and it seems to happen regardless of the graph sometimes. I have no clue if this whole issue is something with my computer. I just haven't seen anyone else experience this exact issue.

**Comment by Pingore** [comment]
I can also provide a video occurring in headset but the graph shows where the lag occurs pretty accurately.

**Comment by DJJolty257** [comment]
maybe this is similar but I have some sort of intense stutter every once in a while - it's just on the server end, the game continues to run while the server dies for a few moments, then everything goes back to normal. Happens at least once every session. It's not detrimental, just immersion-breaking when it occurs.

**Comment by Pingore** [comment]
I have also figured out that if the refresh rate is less than/equal to 72hz the lag spikes increase, but if it is greater than 72hz the lag spikes decrease over time and create gaps. I have tried many configurations (using the nightly with the new settings) and this behavior never changes. This is an example of what it does at 80hz (the first video i posted was at 72hz) :

https://user-images.githubusercontent.com/94215310/198856741-88656dec-b284-4ba2-b7ed-09b61349be7c.mp4

Its pretty much the inverse of what happened before. The end result is still lagging where there are gaps that are more than 2 frames. I will also stress that not a single setting has effected this lag besides refresh rate. I'm open to any suggestions about how to address this but for right now it seems to be a bug.
